Change range from xsd:string to rdf:langString

ckindermann opened the pull request:
This fixes #114

ckindermann commented:
The new commit defines a `mod:string` as the union of `xsd:string` and `rdf:langString` and uses this datatype for `rdf:range`.

jonquet commented:
Shall be integrated I supposed soon. 
One challenge is to maintain "MOD description with MOD" within the source file (which is in this case the reason why the ontology is in fact not consistent !) hence could you : 
- Update    owl:versionInfo let's say to v3.2.1
- Update dcterms:modified   to today

ckindermann commented:
I updated `owl:versionInfo` and `dcterms:modified` for `mod`. I'm assuming you'd also want the new `mod:string` to have additional properties such as `dcterms:description`. Anything else?

jonquet commented:
@ckindermann Thanks for the changes and the contribution. Part 1 is addressed ;)
